packages/medusa/src/loaders/index.ts

```diff
@@ -138,6 +139,7 @@ export default async ({
     onApplicationPrepareShutdown: medusaAppOnApplicationPrepareShutdown,
   } = await loadMedusaApp({
     container,
+    customLinkModules: await resolvePluginsLinks(plugins, container),
```

Comment:
Should this happen inside `loadMedusaApp` instead? 

Also, I don't think we should make a difference between custom and non-custom link definitions, we should load all of them the same way IMO

Reply:
they are all loaded the same way, it is just naming, I can rename it

Reply:
I choose not to do it in medusa app because then it becomes suddenly aware of the modules, I would like to keep this concept outside IMO

Reply:
I ve changed to link modules, but since we also have custom modules, it might make more sense to revert to customLinkModules